shared/ArchiveView.js

```diff
@@ -2,22 +2,70 @@
 
 const {
   TemplateView,
+  AvatarView,
   RoomView,
   RightPanelView,
   LightboxView,
   viewClassForTile,
 } = require('hydrogen-view-sdk');
 
+class RoomHeaderView extends TemplateView {
+  render(t, vm) {
+    return t.div({ className: 'RoomHeader middle-header' }, [
+      t.view(new AvatarView(vm, 32)),
+      t.div({ className: 'room-description' }, [t.h2((vm) => vm.name)]),
+      t.button(
+        {
+          className: 'button-utility room-header-change-dates-button',
+          'aria-label': vm.i18n`Change dates`,
+          onClick: (/*evt*/) => {
+            vm.openRightPanel();
+          },
+        },
+        [
+          // Calendar icon (via `calendar2-date` from Bootstrap)
```

Comment:
Are we ok to use this icon from Bootstrap? https://icons.getbootstrap.com/

I think so based on the MIT license:

> Currently [v1.9.1](https://github.com/twbs/icons/releases/tag/v1.9.1). Icons and documentation licensed [MIT](https://github.com/twbs/icons/blob/main/LICENSE.md).

Reply:
We could also copy the one from the Chrome user-agent style sheet for `input[type="date" i]::-webkit-calendar-picker-indicator`. From the [CSS source](https://github.com/chromium/chromium/blob/d6f46a00b8c3110e7fcadcff78d58ae86c445ec2/third_party/blink/renderer/core/html/resources/html.css#L1200) and [SVG image source](https://github.com/chromium/chromium/blob/d6f46a00b8c3110e7fcadcff78d58ae86c445ec2/third_party/blink/renderer/core/html/resources/images/calendar_icon.svg). But not sure about the [license](https://github.com/chromium/chromium/blob/d6f46a00b8c3110e7fcadcff78d58ae86c445ec2/LICENSE) 🤔

```html
<svg xmlns="http://www.w3.org/2000/svg" width="16" height="15" viewBox="0 0 24 24">
	<path fill="#00000" d="M20 3h-1V1h-2v2H7V1H5v2H4c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 18H4V8h16v13z"/>
	<path fill="none" d="M0 0h24v24H0z"/>
</svg>
```

Reply:
Chatted in the Design room about this, https://matrix.to/#/!zzGPdcyFqxIOJEijoI:matrix.org/$Hm1DOAg6rSkZU1CJlkYnV23miAsKPpZFYHhUwbX2PV0?via=matrix.org&via=element.io&via=envs.net

The license from Bootstrap should be compatible with Apache-2.0 which this project will inherit just like the other `matrix-org` projects -> https://github.com/matrix-org/matrix-public-archive/pull/55 ✅